ui/app/src/state/epics/content.ts

```diff
@@ -98,7 +102,7 @@ import { AjaxError } from 'rxjs/ajax';
 import { showErrorDialog } from '../reducers/dialogs/error';
 import { dissociateTemplate } from '../actions/preview';
 import { isBlank } from '../../utils/string';
-import { DetailedItem } from '../../models';
+import { DetailedItem, SandboxItem } from '../../models';
```

Comment:
Doesn't look like SandboxItem is in use. 

(Build & review for warnings always before final push)